src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.iOS.cs

```diff
@@ -456,5 +455,56 @@ public Rect MakeVisible(UIElement visual, Rect rectangle)
 			ScrollViewExtensions.BringIntoView(this, visual, BringIntoViewMode.ClosestEdge);
 			return rectangle;
 		}
+
+		#region Touches
+
+		/*
+		 * By default the UIScrollView will delay the touches to the content until it detects
+		 * if the manipulation is a drag. And even there, if it detects that the manipulation
+		 * is a Drag, it will cancel the touches on content and handle them internally
+		 * (i.e. Touches<Began|Moved|Ended> will no longer be invoked on SubViews).
+		 * cf. https://developer.apple.com/documentation/uikit/uiscrollview
+		 *
+		 * The "TouchesManager" give the ability to any child UIElement to alter this behavior
+		 * if it needs to handle the gestures itself (e.g. the Thumb of a Slider / ToggleSwitch).
+		 *
+		 * On the UIElement this is defined by the ManipulationMode.
+		 */
+
+		internal UIElement.TouchesManager TouchesManager { get; }
+
+		private void UpdateDelayedTouches()
+		{
+			if (TouchesManager.Listeners == 0)
+			{
+				// This prevents unnecessary touch delays (which affects the pressed visual states of buttons) when user can't scroll.
+				var canScrollVertically = VerticalScrollMode != ScrollMode.Disabled && ContentSize.Height > Frame.Height;
+				var canScrollHorizontally = HorizontalScrollMode != ScrollMode.Disabled && ContentSize.Width > Frame.Width;
+				DelaysContentTouches = canScrollHorizontally || canScrollVertically;
+			}
+			else
+			{
+				DelaysContentTouches = false;
```

Comment:
I think we've left `DelaysContentTouches` as `true` in the past because it represents 'expected' behaviour for the platform. This should probably be flagged as a breaking change if it will vary now depending on the ScrollViewer content.

Reply:
Added in the documentation https://github.com/unoplatform/uno/pull/1256/files#diff-22bbf8f60bf82453e06800aaa19bed33R189